.changeset/big-ravens-turn.md

```diff
@@ -0,0 +1,5 @@
+---
+"astrojs-compiler-sync": major
```

Comment:
I believe this change is not a breaking change, just to provide type information correctly, so I'll make it a minor change.